google/cloud/functions/http_request.h

```diff
@@ -102,7 +102,7 @@ class HttpRequest {
     version_minor_ = minor;
     return *this;
   }
-  HttpRequest&& set_version(int major, int minor) && {
+  HttpRequest&& set_version(int major, int minor) && {  // NOLINT
```

Comment:
nit: was this a `bugprone-easily-swappable-parameters`? If so, the `NOLINT` is superfluous.

Reply:
Done.